spec/planeLanded.spec.js

```diff
@@ -0,0 +1,67 @@
+const TestingFramework = require('./testing-framework');
+const Airport = require('../src/airport');
+const Plane = require('../src/planes');
+
+const testingFramework = new TestingFramework;
+
+let actual, expected, result, plane, airport;
+
+console.log('RUNNING TEST SUITE G: FOR A LANDED PLANE MUST BE AT AN AIRPORT');
+console.log('💻 ~file: planeLanded.spec');
+console.log('++++++++++++++++++++++++++++++++++++++++++++++++++++++++');
+console.log('');
+
+console.log('Test 1G - if a plane lands at an airport it sets plane landed to true');
```

Comment:
Test files are ugly.  Would have rewritten with Jasmine knowing how much quicker it is.